test/example/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress
+spec:
+  ingressClassName: nginx-example
+  rules:
+  - host: localhost
```

Comment:
This cannot work - this requires `Host: localhost` in the HTTP request
served by the ingress, which means request like:

    curl http://localhost/

This will not reach the cluster so it cannot work. We don't have a DNS name for
the clusters, and setting one will be very bad since it requires root, so 
we really don't want to use host based routing. We can avoid this by not
specifying a host like this:

    rules:
    - http:
        paths:
        ...

The ingress will look like:

    $ kubectl get ingress --context ex1
    NAME              CLASS   HOSTS   ADDRESS           PORTS   AGE
    example-ingress   nginx   *       192.168.122.236   80      30m